Gemfile

```diff
@@ -1,12 +1,13 @@
-ruby "2.2.4"
+ruby RUBY_VERSION
```

Comment:
Aha! That helps. Is setting that option really useful here, given that we're only testing against one Ruby and are committing our `Gemfile.lock`? (perhaps a silly question)